au/code/au/chrono_interop_test.cc

```diff
@@ -36,6 +46,36 @@ TEST(DurationQuantity, InterconvertsWithIndirectlyEquivalentChronoDuration) {
     EXPECT_THAT(from_chrono, SameTypeAndValue(seconds(1.234)));
 }
 
+TEST(DurationQuantity, EquivalentOfChronoNanosecondsHasNsLabel) {
+    constexpr auto from_chrono_ns = as_quantity(std::chrono::nanoseconds{123});
+    EXPECT_EQ(stream_to_string(from_chrono_ns), "123 ns");
```

Comment:
Any reason not to use
```suggestion
    EXPECT_STREQ(stream_to_string(from_chrono_ns), "123 ns");
```
or
```suggestion
    EXPECT_THAT(stream_to_string(from_chrono_ns), StrEq("123 ns"));
```
?

Reply:
No special reason; I'll make the switch.  Thanks!